services/distribution/src/main/java/app/coronawarn/server/services/distribution/objectstore/client/S3ClientWrapper.java

```diff
@@ -47,8 +49,16 @@ public class S3ClientWrapper implements ObjectStoreClient {
 
   private final S3Client s3Client;
 
+  private final String dccListPath;
+
   public S3ClientWrapper(S3Client s3Client) {
     this.s3Client = s3Client;
+    this.dccListPath = null;
+  }
+
+  public S3ClientWrapper(final S3Client s3Client, final String dccListPath) {
```

Comment:
@hilmarf on this level of abstraction, shouldn't this rather be exposed as `delimiter` instead of `dccListPath`? I'm sure future me will wonder why we need `dccListPath` here on this level. Btw, I would suggest to rename this to something that reflects that it's about _DCC revocation_

Reply:
added now more JavaDoc